core/jazz_create-serverless-service/swagger/swagger.json

```diff
@@ -285,6 +285,25 @@
 				"require_internal_access": {
                     "type": "boolean",
                     "description":"Does service require access to internal SLF network and resources"
+                },
+                "deployment_targets": {
+                    "type": "object",
+                    "description": "Deployment targets list for the service",
+                    "required": true,
```

Comment:
@devsatishm we have it here too

Reply:
These are merges of someone else's changes, so I can't speak to their reasoning.